test/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/connectors/mocks/MockIncomeTaxSubscriptionConnector.scala

```diff
@@ -44,11 +45,16 @@ trait MockIncomeTaxSubscriptionConnector extends UnitTestTrait with MockitoSugar
     )(ArgumentMatchers.any(), ArgumentMatchers.any())).thenReturn(Future.successful(response))
   }
 
-
   def mockSaveSelfEmployments[T](id: String, value: T)
                                 (response: PostSelfEmploymentsResponse): OngoingStubbing[Future[PostSelfEmploymentsResponse]] = {
     when(mockIncomeTaxSubscriptionConnector.saveSelfEmployments[T](
       ArgumentMatchers.eq(id), ArgumentMatchers.eq(value)
     )(ArgumentMatchers.any(), ArgumentMatchers.any())).thenReturn(Future.successful(response))
   }
+
+  def mockBusinessGetSelfEmployments(id: String)(response: Future[Either[GetSelfEmploymentsFailure, Option[BusinessNameModel]]]): OngoingStubbing[Future[GetSelfEmploymentsResponse[BusinessNameModel]]] =
+    when(mockIncomeTaxSubscriptionConnector.getSelfEmployments[BusinessNameModel](ArgumentMatchers.eq(id))(ArgumentMatchers.any(), ArgumentMatchers.any())).thenReturn(response)
+
+  def mockBusinessSaveSelfEmployments[T](id: String, data: T)(response: Future[Either[PostSelfEmploymentsFailure, PostSelfEmploymentsSuccess]]): OngoingStubbing[Future[PostSelfEmploymentsResponse]] =
+    when(mockIncomeTaxSubscriptionConnector.saveSelfEmployments(ArgumentMatchers.eq(id), ArgumentMatchers.eq(data))(ArgumentMatchers.any(), ArgumentMatchers.any())).thenReturn(response)
```

Comment:
Could be replaced with the above two functions as they are generic and can do what these two functions do